File: Assets/ShootingRPGKit/Scripts/EnemyScript/WeakPoint.js
#pragma strict
var master : Transform;
var damageMultiply : float = 2;
var isCritical : boolean = true;
var ignoreGuard : boolean = true;

function Start (){
	if(!master){
		master = transform.root;
	}
	this.tag = "WeakPoint";
}

File: Assets/ShootingRPGKit/CsharpExample/ScriptCS/EnemyScript/WeakPoint.cs
using UnityEngine;
using System.Collections;

public class WeakPoint : MonoBehaviour {
	public Transform master;
	public int damagePercent = 200;
	public bool isCritical = true;
	public bool ignoreGuard = true;
	// Use this for initialization
	void Start () {
		if(!master){
			master = transform.root;
		}
		this.tag = "WeakPoint";
	}

}
